Handbook/Handbook.md

```diff
@@ -133,6 +133,19 @@ Data Department staff are often working on multiple projects simultaneously. Thi
 * **Low-Context Communication:** Assume that the person you are communicating with does not know the context of your work and/or question. A simple question like 'What's the difference between `AS_HEADT` and `AS_HEADBR`?' actually has a very lengthy response, depending on the context of the question. In order to make communication efficient, try signaling the context of the question. E.g. "I am trying to report 'final' assessed values for 2019 to compare to 'initial' assessed values. Which HEAD file should I use for each set?" Take the extra time to contextualize your question.
 * **Asynchronous Communication:** In general, each team member should not expect other team members to respond quickly to questions. This is due to the variety and timing of the work we do. When you hit a stumbling block and need to ask a clarifying question, do so, and then work on a different task while you wait for a response. Similarly, unless otherwise indicated, do not interpret questions from other team members as requiring an immediate response.
 
+## Scheduling time off
+
+When scheduling expected time off (appointments, vacations, etc.), please use
+the process below to block your time from scheduling and let others know when
+you're out. For unexpected time off (sick, emergencies), your supervisor will
+complete the process for you.
+
+1. Use [CCT](https://www.cookcountyil.gov/cct) to request your time off, as outlined in Section IV of the [CCAO Employee Handbook](http://intranet/filedownloads/EmployeeHandbook.pdf).
+2. Once your CCT request is approved, add the time off to the `Assessor.Data` shared calendar using the following steps:
+    * Create a calendar appointment with the title `$NAME OOO - $REASON`, where `$NAME` is your own first name and `$REASON` is the reason you're out-of-office. It doesn't have to be specific, i.e. just "Appointment" or "Vacation" is sufficient.
+    * Change the appointment type (the dropdown next to `Busy`) to `Out of office`. This changes the coloring/behavior of event. If you're out the entire day, use the `All day` event time.
+    * Invite the Data calendar using the `Assessor.Data@cookcountyil.gov` email address. This will share the out-of-office event with others in the Department.
+
```

Comment:
Let's also add something like the below for execs:

3. For  Directors and Deputy employees only: if you are taking a full workday off or more, please also use a duplicate calendar invitation to notify other CCAO Executives. The steps are below.
* Create a duplicate event of your time off with the title `$NAME OOO`.
* Change the appointment type to `Free`. This changes the coloring/behavior of the event as it will appear on other Execs calendars. 
* To the attendees list, as Optional, add `CCAO Executives` 
* Suggested but not required are two options that will then appear as highlighted below.
![image](https://github.com/ccao-data/wiki/assets/138712135/58fcda4b-beca-432b-b54a-1fa5ae46d3fc)

* Shown in green: Outlook may try to create a Teams meeting because you've invited others. If it does this, click `❌ Don't Host Online` to remove the Teams meeting. 
* Shown in blue: The icon with a purple arrow is  `Response Options` menu. Click it and de-select `Request Responses`. De-selecting this will mean that the recipients of this event can choose whether to show it on their calendar, but you won't get an email response from every single one.

Reply:
Great suggestion! Done in d8857f7